scrapers/scrape_fl.py

```diff
@@ -28,3 +28,16 @@
     dd.recovered = int(dd.cases) - int(dd.deaths)
 
 print(dd)
+
+# get the data from PDF file containing full history
+history_url = 'https://www.llv.li/files/ag/aktuelle-fallzahlen.pdf'
+d = sc.pdfdownload(history_url, layout=True, silent=True)
+for row in d.splitlines():
+    m = re.search(r'^((?:Montag|Dienstag|Mittwoch|Donnerstag|Freitag|Samstag|Sonntag).+\d{4})\s+(\d+)$', row)
```

Comment:
You should remove the weekday ("Montag, ", "Dienstag, ") from the first group, since the date can't be parsed like that:

```
FL Error: Unknown date/time format: Montag, 29. Juni 2020
Traceback (most recent call last):
  File "./scrapers/parse_scrape_output.py", line 165, in <module>
    new_date = parse_date(v)
  File "/home/odi/dev/covid_19/scrapers/scrape_dates.py", line 177, in parse_date
    assert False, f"Unknown date/time format: {d}"
AssertionError: Unknown date/time format: Montag, 29. Juni 2020
```

Reply:
Done